plugin.video.sgtv/resources/search.xml

```diff
@@ -0,0 +1,2 @@
+(lp0
```

Comment:
Not sure what is the search.xml. Does it really need to be shipped with the addon? This is probably best placed in the addon userdata